### PR TITLE
doc: update documentation on Yarn 2 usage

### PR DESCRIPTION
**Pull request by soofstad**
Add some documentation and extra check on yarn install in Dockerfile

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -153,4 +153,5 @@ web/.yarn/*
 !web/.yarn/plugins
 !web/.yarn/releases
 !web/.yarn/sdks
-!web/.yarn/versions
\ No newline at end of file
+!web/.yarn/versions
+/web/node_modules/
```
Comment (Ingerid-Bouvet) [inline .gitignore:157]: Already excluded on line 10

File: web/Dockerfile [hunk 2]
```diff
@@ -16,15 +16,15 @@ ENV REACT_APP_AUTH_REDIRECT_URI=$REDIRECT_URI
...
+RUN yarn install --immutable --immutable-cache
 
 FROM base as development
 CMD ["yarn", "start"]
 
 FROM base as build
 RUN yarn build
 
-FROM node:12-alpine as prod
+FROM node:14-alpine as prod
```
Comment (Ingerid-Bouvet) [inline web/Dockerfile:27]: Any reason why we're not using `node:18-alpine`?

**Review by Ingerid-Bouvet — CHANGES_REQUESTED** [review]
Nice! I planned to document this myself, but I wanted to wait until the PR got through so I didn't write documentation for something we might not end up using. But I couldn't have included a piece about Jetbrains, so good that you did that.

File: web/Dockerfile (hunk 2 above)
Comment (soofstad) [reply web/Dockerfile:27]: no

**Review by Ingerid-Bouvet — APPROVED** [review]

**Pull request merged**